### PR TITLE
Nostr NIP-60

### PR DESCRIPTION
**Pull request by davidcaseria**
[NIP-60](https://github.com/nostr-protocol/nips/pull/1369)

This implementation takes a hybrid approach: the wallet info is stored as a kind `37375` event but keeps the proofs stored locally. A CDK application developer can optionally save the `7375` events by providing a `TxInfo` object for transactions. If a wallet should be accessible on another device or application, the restore endpoint for each mint should be called to retrieve any missing proofs.

### CODE REVIEW

**Review by thesimplekid — COMMENTED** [review]
What do you think of making this a separate crate like we do for the sqlite and redb. This is a bit of a smaller implantation then those require so maybe just a feature flag makes sense, but may be best to stick to that pattern?

**Comment by davidcaseria** [comment]
Sounds good. I will work on that this week.

File: crates/cdk/src/mint_url.rs [hunk 1]
```diff
@@ -91,6 +91,12 @@ impl fmt::Display for MintUrl {
     }
 }
 
+impl From<Url> for MintUrl {
+    fn from(url: Url) -> Self {
+        Self(url.to_string())
```
Comment (ok300) [inline crates/cdk/src/mint_url.rs:96]: IMO it's better to use `MintUrl::from_str` here because it does some extra validation and processing.

**Comment by davidcaseria** [comment]
This is not compatible with the merged version NIP-60, so closing for now.

**Pull request closed without merging**